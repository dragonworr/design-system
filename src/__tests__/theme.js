import {
  theme,
  colors,
} from '..'

describe('theme', () => {
  test('exports an object', () => {
    expect(typeof theme).toBe('object')
  })

  test('exports colors', () => {
    expect(typeof colors).toBe('object')
  })
})
