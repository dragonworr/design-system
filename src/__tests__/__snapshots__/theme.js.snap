// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme snapshot 1`] = `
Object {
  "bold": 600,
  "boxShadows": Array [
    "0 0 4px 0 rgba(0,0,0,0.08)",
    "0 8px 8px 0 rgba(0,0,0,0.08)",
    "0 16px 16px 0 rgba(0,0,0,0.08)",
    "0 32px 32px 0 rgba(0,0,0,0.08)",
  ],
  "breakpoints": Array [
    32,
    40,
    48,
    64,
  ],
  "colors": Object {
    "black": "#000",
    "blue": "#007aff",
    "blue0": "#cdf",
    "blue1": "#cdf",
    "blue2": "#007aff",
    "blue3": "#007aff",
    "blues": Array [
      "#cdf",
      "#cdf",
      "#007aff",
      "#007aff",
    ],
    "borderGray": "#d1d6db",
    "darkBlue": "#049",
    "darkGray": "#364049",
    "darkGreen": "#060",
    "darkOrange": "#950",
    "darkPurple": "#407",
    "darkRed": "#800",
    "gray": "#687B8E",
    "gray0": "#f0f2f4",
    "gray1": "#f0f2f4",
    "gray2": "#687B8E",
    "gray3": "#687B8E",
    "green": "#0a0",
    "green0": "#cec",
    "green1": "#cec",
    "green2": "#0a0",
    "green3": "#0a0",
    "greens": Array [
      "#cec",
      "#cec",
      "#0a0",
      "#0a0",
    ],
    "lightBlue": "#cdf",
    "lightGray": "#f0f2f4",
    "lightGreen": "#cec",
    "lightOrange": "#fec",
    "lightPurple": "#ecf",
    "lightRed": "#fcc",
    "orange": "#f90",
    "orange0": "#fec",
    "orange1": "#fec",
    "orange2": "#f90",
    "orange3": "#f90",
    "oranges": Array [
      "#fec",
      "#fec",
      "#f90",
      "#f90",
    ],
    "purple": "#70b",
    "purple0": "#ecf",
    "purple1": "#ecf",
    "purple2": "#70b",
    "purple3": "#70b",
    "purples": Array [
      "#ecf",
      "#ecf",
      "#70b",
      "#70b",
    ],
    "red": "#c00",
    "red0": "#fcc",
    "red1": "#fcc",
    "red2": "#c00",
    "red3": "#c00",
    "reds": Array [
      "#fcc",
      "#fcc",
      "#c00",
      "#c00",
    ],
    "text": "#001833",
    "white": "#fff",
  },
  "font": "'Montserrat','Helvetica Neue',Helvetica,Arial,sans-serif",
  "fontSizes": Array [
    12,
    14,
    16,
    20,
    24,
    32,
    48,
  ],
  "fontWeights": Object {
    "bold": 600,
    "regular": 400,
  },
  "maxContainerWidth": "1280px",
  "mediaQueries": Array [
    "@media screen and (min-width:32em)",
    "@media screen and (min-width:40em)",
    "@media screen and (min-width:48em)",
    "@media screen and (min-width:64em)",
  ],
  "radii": Array [
    0,
    2,
    6,
  ],
  "radius": "2px",
  "regular": 400,
  "space": Array [
    0,
    4,
    8,
    16,
    32,
    64,
    128,
  ],
}
`;
